pkg/apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -197,6 +197,26 @@ type PipelineSpec struct {
 	//
 	// +optional
 	Name string `json:"name,omitempty"`
+
+	// Parse enables parsing of log entries into structured logs
+	//
+	// Logs are parsed according to parse value, only `json` is supported as of now.
+	//
+	// +kubebuilder:validation:Enum:=json
+	// +kubebuilder:default:=json
```

Comment:
No default, it is optional but if present the only allowed value is "json"
```suggestion
```

Reply:
changed